### PR TITLE
Add Edge-LB dashboards

### PR DESCRIPTION
**Pull request by vespian**
## High-level description (required)

This PR adds the following dashboards for Edge-LB:
* Go runtime overview
* Go runtime memory usage details
* Haproxy metrics

The HAProxy has a separate LICENSE file as we are extending an LGPL-licensed dashboard created by @rfrail3 [1]. CC: @mattj-io There is also licensing information along with the attribution of the author of the original dashboard at the bottom of the `Edge-LB_ Pool server - HAProxy.json` itself.

[1] https://github.com/rfrail3/grafana-dashboards/blob/master/prometheus/haproxy-full.json

## Related Jiras:

- https://jira.mesosphere.com/browse/DCOS-54002 `Create Grafana Dashboards for all the goruntime-specific data`
- https://jira.mesosphere.com/browse/DCOS-54003 `Create Grafana Dashboards for all the haproxy metrics`

### CODE REVIEW

**Comment by vespian** [comment]
@mattj-io Do we need to modify the root folder's license file somehow (i.e. it's Apache 2.0) so that it mentions that individual dashboards can have a different license, or is having the separate folder plus a note in the dashboard itself enough?

**Comment by vespian** [comment]
@philipnrmn @gracedo @branden Could you please have a look? The separate licensing was already discussed with Matt and Nathan.

File: dashboards/Edge-LB/Edge-LB_ Pool server - HAProxy/Edge-LB_ Pool server - HAProxy.json [hunk 1]
```diff
@@ -0,0 +1,4981 @@
...
+            "y": 25
+          },
+          "id": 29,
+          "legend": {
+            "alignAsTable": true,
+            "avg": true,
+            "current": true,
+            "max": true,
+            "min": true,
+
```
Comment (gracedo) [inline dashboards/Edge-LB/Edge-LB_ Pool server - HAProxy/Edge-LB_ Pool server - HAProxy.json]: nit: can we capitalize `Server`?

**Review by gracedo — CHANGES_REQUESTED** [review]
Can you remove the spaces from these filenames? Otherwise lgtm

**Comment by vespian** [comment]
@gracedo Done, ready for next pass!

Could you also have a look at the backport, please? 

https://github.com/dcos/grafana-dashboards/pull/74

**Review by gracedo — APPROVED** [review]

**Pull request merged**